CHANGELOG.md

```diff
@@ -10,9 +10,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Fix editing of async voice messages [#2943](https://github.com/GetStream/stream-chat-swift/pull/2943)
 
 ## StreamChatUI
+### ✅ Added
+- Makes it easier for custom attachments to support mixed attachments rendering [#2947](https://github.com/GetStream/stream-chat-swift/pull/2947)
```

Comment:
maybe rephrase this? "makes it" - what's "it"? 
Maybe "Better support for mixed attachments rendering"?

Reply:
Done ✅